Sources/Latte.swift

```diff
@@ -36,6 +51,7 @@ public struct LatteWebViewResult {
 
 public enum LatteWebViewEvent {
     case openEmailClient
+    case viewPage(path: String)
```

Comment:
```swift 
case viewPage(event: LattePageViewEvent)
```

Reply:
1. Instead of `LattePageViewEvent`, I call it `LatteViewPageEvent`. Since we had `openEmailClient`, for consistency, calling it `viewPage` is better? Thought? 
2. In dbaeb2e, I kept the delegate function as `func latte(onViewPage _: Latte, path: String)`. Do you think it is better to use the event object here as well? like `func latte(onViewPage _: Latte, event: LattePageViewEvent)`